### ISSUE TITLE
reopening the browser should not restore scroll position

### ISSUE THREAD

**Issue by shalva97** [minbrowser/min#1301]
* **Min Version:** 1.16.0
* **Operating system:**  Arch

## Expected Behavior
when i scroll Reddit, then reopen browser, scroll should be on the top

## Actual Behavior
it still remembers scroll position and starts to load bunch of posts to be able to scroll to bottom

## To Reproduce
scroll some website and reopen the browser

**Comment by PalmerAL** [comment]
This is interesting, because in most cases I think it actually makes sense to restore the scroll position (for things like news articles, for example). Ideally, you could make an exception for infinite-scrolling sites, but I don't see an easy way to do that at the moment.

**Comment by shalva97** [comment]
Yeah, on some websites it is useful and now it willbe bad if this gets disabled. 
Is sit possible to save the scroll height and if it is less on borwser restart then dont restore?

**Comment by shalva97** [comment]
I just noticed that scrolling on reddit and then reloading with ctrl+r does not restore scroll position, but on few other websites it does restore the scroll position, for example on wikipedia

**Comment by PalmerAL** [comment]
Restoring on reload is controlled by Chromium; restoring on restart is controlled by Min (confusing, I know). So it's two separate code paths.
